scripts/notebook

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${CI}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Launches a Jupyter notebook.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+        jupyter notebook \
+            --ip=0.0.0.0 \
+            --port=8888 \
+            --allow-root # Useful if run within the Docker containers
```

Comment:
It's probably very minor, but can we add an argument to the script to opt-in to `--allow-root`, and use that flag in `scripts/docker`? I don't know enough to know that this is a security hole, but an implicit `--alow-root` smells funny. Feel free to tell me I'm wrong and this isn't an issue at all :-D.

Reply:
Yeah, fair enough. I'm not enthralled that the Docker containers are running as root, but it can result in wonky permissions problems when mounting local volumes.
The use-case I was considering for running in the Docker containers doesn't actually make sense here, so I'll just remove it.